### DISCUSSION TITLE
approveChatJoinRequest

### DISCUSSION THREAD

**Discussion by codexJoin** [gram-js/gramjs#345] [Q&A] [answered]
I didn't find in the documentation a way to approve the entry request or to disapprove is it possible?, if yes, can you send me examples?

Telegram have methods approveChatJoinRequest

**Comment by rojvv** [comment, marked as answer]
You can approve like this:

```ts
await client.invoke(
  new Api.messages.HideChatJoinRequest({
    peer: chatId,
    userId: userId,
    approved: true,
  }),
);
```

And disapprove like this:

```ts
await client.invoke(
  new Api.messages.HideChatJoinRequest({
    peer: chatId,
    userId: userId,
    approved: false,
  }),
);
```

**Reply by codexJoin** [reply]
Thx, it's worked!

**Reply by rojvv** [reply]
Remember to mark as answer.

![image](https://user-images.githubusercontent.com/49933115/176021750-04cdbcb5-21e7-48d9-bbc9-8a6d694aedfd.png)
